src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```diff
@@ -309,6 +311,14 @@ Copyright (c) .NET Foundation. All rights reserved.
       <Output TaskParameter="PackageDependencies" ItemName="PackageDependencies" />
     </ResolvePackageAssets>
 
+    <ItemGroup Condition="'$(CopyDebugSymbolFilesFromPackages)' == 'true'">
+      <ReferenceCopyLocalPaths Include="@(_DebugSymbolsFiles)"></ReferenceCopyLocalPaths>
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(CopyDocumentationFilesFromPackages)' == 'true'">
```

Comment:
If `PublishReferencesDocumentationFiles` is true, then we should also set `CopyDocumentationFilesFromPackages` to true, if it isn't already set.

That will also add a few more test cases.

Reply:
This will break many unit-tests https://github.com/dotnet/sdk/pull/27580#issuecomment-1240884830 because
`PublishReferencesDocumentationFiles` is set to true by default.

Reply:
In that case you're right, we shouldn't make the change I suggested.

Probably we should test the publish behavior when `PublishReferencesDocumentationFiles` is false but `CopyDocumentationFilesFromPackages` is true.

Reply:
done in last commit